### DISCUSSION TITLE
Sony head units

### DISCUSSION THREAD

**Discussion by taglia** [ludwig-v/wireless-carplay-dongle-reverse-engineering#117] [Q&A]
Hi, did anyone manage to make the dongle work on Sony head units? I have installed a [XAV-AX7000](https://electronics.sony.com/audio/car-marine-audio/receiver-players/p/xavax7000) on my Harley, and I have tried different versions of the firmware. With some, the dongle keeps resetting; with others, the head unit itself resets itself. I am sure the dongle itself works; I have tried it on a friend’s car. Also, the head unit works fine if I connect my phone to the USB plug. Any ideas or suggestions to help me understand why it does not work would be much appreciated.

Thank you in advance for your help!

**Comment by matteofreddi** [comment]
Hi, did try various adapters with various firmwares on Ax3250, the best I get is video working but no audio at all. (No mic and no speakers working)

**Comment by timmerk** [comment]
Tested and working perfectly with a Sony XAV-AX100 and Sony XAV-AX5000 with stock firmware from 2020 (couldn't tell you the version at the moment.)

**Comment by Converseallstar95** [comment]
I use mine with an XAV-AX5000 (latest firmware v1.06 I think) with no issues. Also latest red UI firmware on the adapter. Try resetting the bluetooth/wifi connections on the adapter and make sure you're plugging into USB 1 (not 2--not sure if that Sony head unit has 2).
